R/grid.R

```diff
@@ -65,13 +65,32 @@ draw_custom_grid <- function(object, view.name){
   grid.args <- remove_field(grid.args, "equilogs")
   at <- list()
   for (side.name in side.names){
-    usr.at <- axis_axTicks(object, as.side(side.name))
-    if (is.null(usr.at))
-      at[[side.name]] <- grid_axTicks(object, as.side(side.name))
-    else
-      at[[side.name]] <- usr.at
+    side.axis <- as.axis(side.name)
+    if(side.axis == "x") {
+      n.side <- grid.args$nx 
+    } else { 
+      n.side <- grid.args$ny
+    }
+    
+    if(is.null(n.side)){
+      usr.at <- axis_axTicks(object, as.side(side.name))
+      if (is.null(usr.at)) {
+        at[[side.name]] <- grid_axTicks(object, as.side(side.name))
+      } else {
+        at[[side.name]] <- usr.at
+      }
+    } else {
+      plot.lims <- lim(object, as.side(side.name))
+      if(is.na(n.side)) {
+        at[[side.name]] <- NULL
+      } else {
+        at[[side.name]] <- seq.int(plot.lims[1], plot.lims[2], length.out = n.side + 1)
```

Comment:
@ldecicco-USGS no, no it does not. Figured this was pretty incomplete. Thanks for pointing that out. Struggling with the best implementation of this function - is there a reason why `grid.gsplot` does not eventually call on `graphics::grid`? That's the pattern we follow for a lot of the other functions. Otherwise, we will be duplicating code. I know the graphics version already handles all of that